citi-onboarding/hacka-pride
# feature(banner): build app to deploy

## What I did
* Fixed animaitions
* Fixed bugs on hamburguer navbar
* Build to deploy
* Changes on keystone admin

## How to test
1. run yarn start and node at the same time
2. verify the animations and mobile navbar
3. verify the access to keystone
